src/constants/endpoint.ts

```diff
@@ -1,4 +1,8 @@
-// Define ENDPOINT 
-export const BASE_ENDPOINT = "/api";
+import urlJoin from 'url-join';
 
-export const SERVER_STATUS_ENDPOINT = BASE_ENDPOINT + "/server-status";
\ No newline at end of file
+// Define ENDPOINT
+export const BASE_ENDPOINT = '/api';
+
+export const SERVER_STATUS_ENDPOINT = BASE_ENDPOINT + '/server-status';
```

Comment:
`status`

Reply:
or `health`

Reply:
@caravinci will it cause any issues elsewhere if i update this endpoint to `/health`?

Reply:
not at all. thanks for checking!